WebRoot/js/zimbraMail/briefcase/model/ZmBriefcaseItem.js

```diff
@@ -359,9 +359,12 @@ function() {
 
 ZmBriefcaseBaseItem.prototype._loadFromDom =
 function(node) {
-
-	this.id = node.id;
-
+	if(node.sfid)  {
+	    this.id = ""+ node.sfid;
+	    this.sfid = ""+node.id;
```

Comment:
Please add comment in code for future reference regarding this swapping ?

Reply:
commented on the swapping of ids